resources/assets/js/app/services/RequestService.js

```diff
@@ -1,13 +1,22 @@
 import axios from 'axios';
 
+const requestParams = {
+    headers: {
+        'Accept': 'application/json',
+    }
+};
+
 export const makeRequest = (method, ...args) => {
     if (typeof axios[method] === 'function') {
+        args.push(requestParams);
+
         return axios[method].apply(this, args);
     }
 };
 
 export const securedRequest = (method, ...args) => {
-    // do something with header (args[2].headers)
+    requestParams.headers['Authorization'] = 'Bearer ' + sessionStorage.jwt;
```

Comment:
ensure that this row won't rewrite requestParams, regular request shouldn't contains Authorization key

create class AuthService or AuthManager that will contains method getToken() and returns JWT token